cmd/satellite/entrypoint

```diff
@@ -3,13 +3,6 @@ set -euo pipefail
 
 SETUP_PARAMS=""
 
-if [[ -n "${API_KEY:-}" ]]; then
-	export STORJ_POINTER_DB_AUTH_API_KEY="${API_KEY}"
```

Comment:
I'm just curious how I can know that this variables aren't used anymore

Reply:
`grep -i api scripts/testdata/satellite-config.yaml.lock` yields only comments. I think this is good.

Reply:
I didn't know that all the configuration options of the `config.yaml` are mapped to env vars with `STORJ_` prefix